client/src/components/Community/Communities/CompetitiveCoding.community.js

```diff
@@ -1,11 +1,13 @@
-import React from 'react';
-import 'bootstrap/dist/css/bootstrap.min.css';
-import CompetitiveCodingMembers from '../../Members/Community/CompetitiveCoding.members';
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import CompetitiveCodingMembers from "../../Members/Community/CompetitiveCoding.members";
+import CommunityTemplate from "./CommunityTemplate";
+import { CommunityDetails as community } from "./CommunityDetails";
 
 function CompetitiveCoding() {
   return (
     <div className="competitive-coding">
-      <h1>This is Competitive coding community page.</h1>
+      <CommunityTemplate Community_details={community.CP} />
       <CompetitiveCodingMembers />
     </div>
   );
```

Comment:
These community components are also not required, Just keep one generic community component and render details based on URL.
Thoughts?

Reply:
Ok now I got it, it would make the file structure more clean and easy to maintain/edit 😃.So we're targeting towards replacing different community/member files with a single file that will render details according to the URL.  It would require changing the routes for the different community and member files and also we'll need to restructure the Member and Community folder also 🤔 . Correct me if I'm wrong. @ankiiitraj @EmperorYP7

Reply:
So, What I'm thinking now is having 1 component and two array files one for community details and another for member details and then we can use these arr files for data in both the components of members and community.

Yup this will require some file structure changes and if possible we should go for this!
inside the community, we can have routes like /community/CP

Reply:
I'll try to implement this in my next commit. 🙌

Reply:
Well, on the member's page we'll be rendering every community's members. File structure needs to be revised. Try to implement this for competitive coding for now @tusharjain0022. For other communities, fill in some dummy text.

Reply:
I've implemented this on the community section only, Now we've only one file `CommunityTemplate.js` that will render the details depending upon the URL, Same can be implemented for the Member section also. For now, I've moved all the unused files to the `to_be_removed` folder.